### PR TITLE
Clean messaging

### PR DESCRIPTION
**Pull request by squirrelo**
Replaces newlines with HTML line breaks and linkifies all message strings.

### CODE REVIEW

File: qiita_pet/util.py [hunk 1]
```diff
@@ -68,6 +69,15 @@ def clean_str(item):
     return str(item).replace(" ", "_").replace(":", "")
 
 
+def convert_text_html(message):
+    """Linkify URLs and turn newlines into <br/> for HTML"""
+    lines = message.split('\n')
```
Comment (josenavas) [inline qiita_pet/util.py]: I think this entire function can be replaced by:
`return tornado_linkify(message.replace('\n', '<br/>'))`

Comment (squirrelo) [inline qiita_pet/util.py]: Replace needs to happen after linkifying, but it works. Done.

File: qiita_pet/handlers/study_handlers/description_handlers.py [hunk 2]
```diff
@@ -211,6 +213,7 @@ def process_sample_template(self, study, user, callback):
                          else 'parsing the sample template')
             msg = html_error_message % (error_msg, basename(fp_rsp),
                                         str(e))
+            msg = convert_text_html(msg)
```
Comment (josenavas) [inline qiita_pet/handlers/study_handlers/description_handlers.py]: I'm going to comment only here but applies to all. There is something weird when passing the `html_error_message` variable contents to the `convert_text_html`, as not all the html is interpreted correctly:
screen shot 2015-08-07 at 2 06 42 pm
Note that the only part of the error message that needs to be passed to `convert_text_html`, which is the variable `e`.
Suggested change:
```python
msg = html_error_message % (error_msg, basename(fp_rsp), convert_text_html(str(e)))
```

Comment (squirrelo) [inline qiita_pet/handlers/study_handlers/description_handlers.py]: Seems linkify also escapes the HTML, interesting. Will fix.

**Comment by squirrelo** [comment]
Change made, good to go.

**Comment by josenavas** [comment]
:+1:

**Pull request merged**